src/previewSupport/prettyPrint.js

```diff
@@ -1,5 +1,7 @@
-import inspect from 'object-inspect';
+import superJson from 'super-json';
+
+const myJson = superJson.create();
 
 export default function prettyPrint(value) {
```

Comment:
Honestly we probably don’t need `prettyPrint` to be its own module (or even its own function) at this point—I would just inline it in `handleConsoleLogs`